src/native/libs/System.Globalization.Native/pal_collation.m

```diff
@@ -117,6 +117,11 @@ int32_t GlobalizationNative_CompareStringNative(const uint16_t* localeName, int3
     }
 }
 
+/**
+ * Removes zero-width and other weightless characters such as U+200B (Zero Width Space), 
```

Comment:
> * Removes zero-width and other weightless characters such as U+200B (Zero Width Space), [](http://example.com/codeflow?start=0&length=88)

just in case if this need to change later, it will be good to look at the full list of ignorable characters [https://www.unicode.org/charts/collation/chart_Ignored.html](https://www.unicode.org/charts/collation/chart_Ignored.html)